[IMP] POS: path to configure payment terminals (#7538)
odoo/documentation: content/applications/sales/point_of_sale/payment_methods/terminals/adyen.rst

   - `Adyen Docs - Payment terminal quickstart guides
     <https://docs.adyen.com/point-of-sale/user-manuals>`_.

Generate an Adyen API key
-------------------------

> The **Adyen API key** is a key used to authenticate your requests. To generate an API key, go to

Reviewer: ```suggestion
The **Adyen API key** is used to authenticate your requests. To generate an API key, go to
```
It is redundant to write key twice

   `/pos_adyen/notification` in the :guilabel:`Event URLs` field; and
#. click :guilabel:`Save` at the bottom of the screen to save the changes.

Configure the payment method
----------------------------

> First, :ref:`enable the payment terminal <terminals/configuration>` and :doc:`create the related

Reviewer: ```suggestion
Start by :ref:`enabling the payment terminal <terminals/configuration>` and :doc:`creating the related
```
If you use "First", there must be a "Second" somewhere, here you only have "First" and "Finally"

> payment method <../../payment_methods>`. Set the journal as :guilabel:`Bank` and select

Reviewer: ```suggestion
payment method <../../payment_methods>`. Set the journal type as :guilabel:`Bank` and select
```
